src/firebase_functions/private/_identity_fn.py

```diff
@@ -114,7 +114,7 @@ def _auth_user_record_from_token_data(token_data: dict[str, _typing.Any]):
     return AuthUserRecord(
         uid=token_data["uid"],
         email=token_data.get("email"),
-        email_verified=token_data.get("email_verified"),
+        email_verified=bool(token_data.get("email_verified")),
```

Comment:
hmm this feels unrelated? should we make a separate PR for this or is this necessary for log impl?

Reply:
mypy was updated and was causing CI to fail because the types didn't match. I'm open to suggestions. This is mainly a linting fix

Reply:
maybe clearest to make the mypy related linting fixes in a separate PR, and have only the logging things here.

Reply:
Without these, the CI doesn't pass. Possibly can make a separate PR for these that can merged into `main`, before we merge this in so that we can have those changes rebased into this branch. I'll look into doing this next week :)

Reply:
eh let's save ourselves some time and push it in this PR.